js/Thread.js

```diff
@@ -1,6 +1,7 @@
-import { NativeModules, DeviceEventEmitter } from "react-native";
+import { NativeModules, NativeEventEmitter } from "react-native";
```

Comment:
wasn't working for me with DeviceEventEmitter, but works with NativeEventEmitter